integration/test_departing_employee.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from integration import run_command
+
+DEPARTING_EMPLOYEE_COMMAND = "code42 departing-employee"
+
+
+@pytest.mark.parametrize(
+    "command, error_msg",
+    [
+        (f"{DEPARTING_EMPLOYEE_COMMAND} add", "Missing argument 'USERNAME'."),
+        (f"{DEPARTING_EMPLOYEE_COMMAND} remove", "Missing argument 'USERNAME'."),
+        (f"{DEPARTING_EMPLOYEE_COMMAND} bulk add", "Missing argument 'CSV_FILE'."),
+        (f"{DEPARTING_EMPLOYEE_COMMAND} bulk remove", "Missing argument 'FILE'."),
+    ],
+)
+def test_departing_employee_command_returns_error_exit_status_when_missing_required_parameters(
```

Comment:
These negative tests should fall under unit tests.

Reply:
Yes I agree. Should we move them to their respective unit test modules? There might some tests of this similar nature dispersed already, but I know there is not a ton